pyspark/bigdl/nn/keras/layer.py

```diff
@@ -303,13 +351,15 @@ class Flatten(KerasLayer):
 
     # Arguments
     input_shape: A shape tuple, not including batch.
+    name: String to specify the name of the layer. Default is None.
```

Comment:
remove this? we would generate a default name in scala, not NOne.

Reply:
I will remove `Default is None`. This doc line is needed so that users can know they can specify this parameter.